pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -478,6 +478,11 @@ type TiKVSpec struct {
 	// EnableNamedStatusPort enables status port(20180) in the Pod spec.
 	// If you set it to `true` for an existing cluster, the TiKV cluster will be rolling updated.
 	EnableNamedStatusPort bool `json:"enableNamedStatusPort,omitempty"`
+
+	// MaxScaleInReplicas configures max scale in replicas for TiKV stores.
+	// Defaults to 1
+	// +optional
+	MaxScaleInReplicas int `json:"maxScaleInReplicas,omitempty"`
```

Comment:
```suggestion
	MaxScaleInReplicas *int `json:"maxScaleInReplicas,omitempty"`
```

Reply:
If users set it to `0` explicitly, then we just skip the scale in operation, WDYT?

Reply:
Sounds would be more generalize, but I'm curious about what's the situation would set 0 be meaningful?

Reply:
Maybe just disable the scale-in operation? It may not be necessary, I just want to distinguish the case where this parameter is not configured and it's configured explicitly to 0.